Task1.2/modules/directoryWatcher.js

```diff
@@ -0,0 +1,21 @@
+const EventEmitter = require("events");
+const fs = require("fs");
+const csvtojson = require("csvtojson");
+
+class DirectoryWatcher extends EventEmitter {
+  
+  checkDirectoryForFiles (sourceFilePath, destinationFilePath) {
+    const file = fs.readdirSync(sourceFilePath)[0];
```

Comment:
If you're adding index, why do you need to create a directory? Why don't you directly refer the name of the file. If you're intending on creating the directory, you should consider making this dynamic.